scraper.py

```diff
@@ -5,25 +5,129 @@
 
 import sys
 import argparse
+import requests
+import re
+from my_html_parser import HTMLScraper
 
 
 def create_parser():
+    '''Creates a shell argument parser and returns the parser object'''
     parser = argparse.ArgumentParser(
         description='Scrape a website for URLs, email addresses and ' +
                     'phone numbers')
     parser.add_argument('http',
                         help='The address of the site to be scraped')
+    parser.add_argument('-d', '--todir',
+                        help='Name of the .txt file to write results ' +
+                        ' (defaults to output.txt)',
+                        default='output.txt',
+                        dest='todir')
     return parser
 
 
+def get_html(web_address):
+    '''Retrieves raw html text from a particular webpage and returns it'''
+    web_address = format_address(web_address)
+    try:
+        r = requests.get(web_address)
+    except Exception as e:
```

Comment:
when prototype with requests, I also like to use `r.raise_for_status()` which throws an exception if you don't get a 200 OK reply